core/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js

```diff
@@ -47,17 +47,19 @@ oppia.constant('HUMAN_READABLE_SUBSCRIPTION_SORT_BY_KEYS', {
 });
 
 oppia.controller('CreatorDashboard', [
-  '$scope', '$rootScope', '$http', '$window', 'DateTimeFormatService',
-  'AlertsService', 'CreatorDashboardBackendApiService',
+  '$scope', '$rootScope', '$http', '$uibModal', '$window',
+  'DateTimeFormatService', 'AlertsService', 'CreatorDashboardBackendApiService',
   'RatingComputationService', 'ExplorationCreationService',
+  'QuestionObjectFactory', 'TopicsAndSkillsDashboardBackendApiService',
   'UrlInterpolationService', 'FATAL_ERROR_CODES',
   'EXPLORATION_DROPDOWN_STATS', 'EXPLORATIONS_SORT_BY_KEYS',
   'HUMAN_READABLE_EXPLORATIONS_SORT_BY_KEYS', 'SUBSCRIPTION_SORT_BY_KEYS',
   'HUMAN_READABLE_SUBSCRIPTION_SORT_BY_KEYS',
   function(
-      $scope, $rootScope, $http, $window, DateTimeFormatService,
+      $scope, $rootScope, $http, $uibModal, $window, DateTimeFormatService,
```

Comment:
Line breaks here should match those in the stringified list.

Reply:
Done